subsys/bluetooth/services/ras/rrsp/ras_rrsp.c

```diff
@@ -0,0 +1,5 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
```

Comment:
and this

Reply:
they should be part of the same commit, there is no reason to add a cmake and Kconfig file with a completely empty c file, can you move the cmake and Kconfig the commit with the code or just merge the 2 commits togehter?

Reply:
Merged the commits